tests/legacy-ray_charles.e2e.spec.js

```diff
@@ -54,6 +54,12 @@ for (let i = 0; i < 15; i++) {
 }
 console.log('\nLoading archive:\n' + rayCharlesAllParts + '\n');
 
+/**
+ *  Run the tests
+ * @param {WebDriver} driver - Selenium WebDriver object
+ * @param {array} modes - Array of modes to run the tests in
+ * @returns {undefined}
+*/
```

Comment:
Please format this JSDoc comment like the way I suggested for the Gutenberg test.

Reply:
You don't appear to have formatted it as suggested. I suggested you should mark it as returning a Promise (and ensure it does return any `await driver.quit()` statements).